src/core/api/payment_manager_api/pm-per-nodo/v2/wisp-sendpaymentresult.xml.tpl

```diff
@@ -21,11 +21,7 @@
     <base />
 
     <set-variable name="transactionId" value="@(context.Request.MatchedParameters["transactionId"])" />
-    <set-variable name="backend-base-url" value="@($"{{pm-host}}/pp-restapi-CD/v2")" />
-    <set-variable name="ecommerce_url" value="${ecommerce_ingress_hostname}" />
-    <set-variable name="body_value" value="@(context.Request.Body.As<string>(preserveContent: true))" />
-    <set-backend-service base-url="@((string)context.Variables["backend-base-url"])" />
-
+    <set-backend-service base-url="@("https://${ecommerce_ingress_hostname}/pagopa-ecommerce-transactions-service/")" />
```

Comment:
please remove the `transactionId` unused variable too

Reply:
Done here beaf91b539745b1d50a011cb4e6cbf8c018a580f